book/book.css

```
@@ -0,0 +1,420 @@
+:root {
+  --book-bg: #fbfaf3;
+  --book-paper: #fffdf6;
+  --book-ink: #171717;
+  --book-soft: #55524b;
+  --book-muted: #77736a;
+  --book-rule: rgba(23, 23, 23, 0.18);
+  --book-blue: #3553ff;
+  --book-blue-soft: rgba(53, 83, 255, 0.1);
+  --book-code: #f0ede2;
+  --book-serif: Georgia, "Iowan Old Style", "Source Serif Pro", serif;
```

Stylelint (17.11.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)